src/Components/LandingPage/sections/MostPickedSection.tsx

```diff
@@ -0,0 +1,195 @@
+import img1 from '../../../assets/download.png'
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { Box, FormControl, Grid, TextField } from '@mui/material';
+import { useContext, useEffect, useState } from "react";
+import styles from './MostPickedTaskTest.module.css'
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+import FavoriteBorderOutlinedIcon from '@mui/icons-material/FavoriteBorderOutlined';
+import axios from 'axios';
+import { AuthContext } from '../../../Context/AuthContext';
+import Modal from '@mui/material/Modal';
+import { useForm } from 'react-hook-form';
+import { Link, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+
+
+export default function MostPickedSection() {
+
+  const { userRole }: any = useContext(AuthContext)
+  const [showState, setShowState] = useState();
+  const handleClose = () => setShowState(false);
+  const navigate = useNavigate()
+  const [ads, setAds] = useState();
+
+  const getMostPicked = () => {
+    axios.get("http://154.41.228.234:3000/api/v0/portal/ads", {
+      headers: {
+        Authorization: "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NTlhMjAzNTYzODg0OGJjZTZmMDBmOTciLCJyb2xlIjoidXNlciIsInZlcmlmaWVkIjpmYWxzZSwiaWF0IjoxNzA1NjEyNDgwLCJleHAiOjE3MDY4MjIwODB9.3cdcgYBnwf4VNWPbMJcOn0sMPBtgPqY7_90Bmf5mxqw"
+      }
+    })
+      .then((response) => {
+        setAds(response.data.data.ads)
+        console.log(response.data.data.ads);
+        
+      })
+      .catch((error) => console.log(error))
+  }
+
+  const addToFav = (item: any) => {
+    axios.post(`http://154.41.228.234:3000/api/v0/portal/favorite-rooms`, {
+      roomId: item
+    }, {
+      headers: {
+        Authorization: "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NTlhMjAzNTYzODg0OGJjZTZmMDBmOTciLCJyb2xlIjoidXNlciIsInZlcmlmaWVkIjpmYWxzZSwiaWF0IjoxNzA1NjEyNDgwLCJleHAiOjE3MDY4MjIwODB9.3cdcgYBnwf4VNWPbMJcOn0sMPBtgPqY7_90Bmf5mxqw"
+      }
+    })
+      .then((response) => {
+        toast.success(response.data.data.message);
+      })
+      .catch((error) =>
+      toast.error(error.response.data.message)
+       )
+  }
+
+  const showDetails = (item: any) => {
```

Comment:
You can change the name of the parameter to a more meaningful name such as `roomId`.